wes_client/client.py

```diff
@@ -0,0 +1,121 @@
+from logging import Logger
```

Comment:
`Logger` doesn't seem to be used here.

In any case, the pattern for using module-level loggers is somewhat different (see, e.g., in [TRS-cli](https://github.com/elixir-cloud-aai/TRS-cli); also see [docs](https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial)):

```py
import logging

logger = logging.getLogger(__name__)
```